packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteFilesControls.tsx

```diff
@@ -61,16 +62,17 @@ export const DeleteFilesControls = (props: {
         disabled={disableClose}
       />
       <Title />
-      <ActionStartControl
-        className={`${CLASS_BASE}__delete-files-action-start`}
-      />
-      <ActionCancelControl
-        className={`${CLASS_BASE}__delete-files-action-cancel`}
-      />
-      <StatusDisplayControl
-        className={`${CLASS_BASE}__action-status-display`}
-      />
-      <DataTableControl className={`${CLASS_BASE}__table`} />
+      <ViewElement className={`${CLASS_BASE}__table-wrapper`}>
```

Comment:
Think i saw this applied directly around the `Table` earlier in the PR?

Reply:
🍕

Reply:
⭐ suck